spec/service_consumers/pact_helper.rb

```diff
@@ -35,7 +39,7 @@ def url_encode(str)
     else
       base_url = "https://pact-broker.cloudapps.digital"
       path = "pacts/provider/#{url_encode(name)}/consumer/#{url_encode(consumer_name)}"
-      version_modifier = "versions/#{url_encode(ENV.fetch('PACT_CONSUMER_VERSION', 'branch-master'))}"
+      version_modifier = "versions/#{url_encode(ENV.fetch('PACT_CONSUMER_VERSION', 'branch-imminence-api-pact-tests'))}"
```

Comment:
This change snuck into 73b3e68ac3413667ae69fb67c2bf0695325ca4ae and should ideally be removed.

Reply:
:thu